requirements.txt

```diff
@@ -5,7 +5,7 @@ PyYAML
 requests
 six >= 1.7
 kazoo==2.2.1
-python-etcd==0.4.3
+python-etcd==0.4.5
```

Comment:
I am aware of some bug fixes, because actually one of them was reported and fixed by me.
Patroni is not using python-etcd as it is, but inherits from `etcd,Client` and override pretty much all methods and properties used to communicate with etcd nodes, i.e.: `machines` and `api_execute`.
That means non of the fixed bugs are hitting us.

I am not against 0.4.5, but probably it would be better to have something like  `python-etcd>=0.4.3` as a requirement.